mantle/platform/api/do/api.go

```diff
@@ -49,6 +49,10 @@ type Options struct {
 	Size string
 	// Numeric image ID, {alpha, beta, stable}, or user image name
 	Image string
+	// Identifies if we are using IPv6 or not.
+	// Currently set by checking if the image used is of custom type
```

Comment:
True, but the code already says that.  "Custom images don't support IPv6" would help explain the reason.  And I'd be tempted to put that comment above the check, not here.